documentation/slate/source/includes/objects.md

```diff
@@ -189,6 +189,12 @@ As "direction" is a [place](#place) , it can be a poi in some data.
 |name|string|Name of the commercial mode|
 |physical_modes|array of [physical_mode](#physical-mode)|Physical modes of this commercial mode|
 
+Commercial modes are close from physical modes, but not normalized and can refer to a brand,
+something that can be specific to a network, and known to the traveler.
+Examples: RER in Paris, Busway in Nantes, and also of course Bus, Métro, etc.
+
+Integrators should mainly use that value for text output to the traveler.
+
```

Comment:
add something like "Integrators should use physical_mode to request Navitia and commercial_mode to build the response

Reply:
You mean like that : https://github.com/CanalTP/navitia/pull/2195/files#diff-d5931b2652047b16f47dfb6e61c21d44R216 ? ;)